publiq-ui/pub-checkbox.stories.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PubCheckbox from './pub-checkbox.vue';
+
+<Meta title="pub-checkbox" component={PubCheckbox} />
```

Comment:
Oh, don't forget to put this in the `Components` group now that that PR was merged.
So:
```suggestion
<Meta title="Components/pub-checkbox" component={PubCheckbox} />
```